src/Tribe/Rewrite.php

```diff
@@ -10,6 +10,20 @@
  * Permalinks magic Happens over here!
  */
 class Tribe__Events__Rewrite extends Tribe__Rewrite {
+
+	/**
+	 * Constant holding the transient key for delayed triggered flush from activation.
+	 *
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 4057198a9dafb852c9710df2e8550625</sub>